lib/tbot/testhelpers/srv.go

```diff
@@ -167,16 +175,23 @@ func MakeBot(t *testing.T, client auth.ClientI, name string, roles ...string) *p
 // - Uses a memory storage destination
 // - Does not verify Proxy WebAPI certificates
 func DefaultBotConfig(
```

Comment:
Maybe go for a less verbose name for this parameter - perhaps just `opts`.

Reply:
fixed